app/build.gradle

```diff
@@ -3,13 +3,18 @@ import groovy.xml.XmlUtil
 apply plugin: 'com.android.application'
 
 android {
+
     compileSdkVersion 28
-    buildToolsVersion '28.0.0'
+    buildToolsVersion '28.0.3'
+
+    defaultConfig {
+        minSdkVersion 26
+        targetSdkVersion 26
```

Comment:
This will likely need to remain in `AndroidManifest.xml`, because when the application is built in the AOSP source tree Gradle is not used.

Reply:
I added it back to the Manifest now.